apps/studio/prisma/generated/generatedTypes.ts

```diff
@@ -24,7 +24,7 @@ export type Navbar = {
 export type Permission = {
   id: GeneratedAlways<number>
   resourceId: number
-  userId: number
+  userId: string
```

Comment:
required because we reverted from using bigint `id` to using `cuid`. 

this change was made because the `starter-kit` auth helper seem to have been written with nullable `ids`